docs/pages/config.mdx

```diff
@@ -48,6 +56,23 @@ export default defineWorld({
     },
   },
   deploysDirectory: "./mud-deploys",
+  modules: [
+    {
+      name: "KeysWithValueModule",
+      artifactPath: "@latticexyz/world-modules/out/KeysWithValueModule.sol/KeysWithValueModule.json",
+      root: true,
+      args: [
+        {
+          type: "bytes32",
+          value: resourceToHex({
+            type: "table",
+            namespace: "app",
+            name: "Tasks",
+          }),
+        },
+      ],
```

Comment:
can we make the default recommendation use `resolveTableId` since it's less complex? and save this for the more advanced usage?
```suggestion
      args: [resolveTableId("Tasks")],
```